src/fields/checkbox.ts

```diff
@@ -0,0 +1,27 @@
+import * as Yup from "yup";
+import { IFieldSchemaBase } from "../schema-generator";
+import { ERROR_MESSAGES } from "../shared";
+import { IFieldGenerator } from "./types";
+
+export interface ICheckboxSchema<V = undefined> extends IFieldSchemaBase<"checkbox", V> {}
```

Comment:
will options be implemented in this PR? (for checkbox, radio and multiselect)

the backend needs to validate that the value matches the options too

Reply:
have added the necessary validation for checkbox, radio and multi-select fields.
also added the select field which I missed out